Header header
float32 dist_center
float32 dist_left
float32 dist_right
float32 rel_angle
bool in_bound